### DISCUSSION TITLE
How do I use Custom Sensors?

### DISCUSSION THREAD

**Discussion by rowra44** [davidrapan/ha-solarman#321] [Q&A] [answered]
Hey
Where am I supposed to implement the Custom Sensors? Are you supposed to create another device definition yaml? If yes, how? Clone the whole repo, create the yaml and then instatt that package?

**Comment by davidrapan** [comment, marked as answer]
It depends:
* You can just copy profile into custom folder directly in your HA and this file will persist through integration updates.
* Or you can create PR, if you think others would benefit from that custom sensor too, and have it added to this repo.
